src/helpers.js

```diff
@@ -1,3 +1,17 @@
+function isNulledBox(box) {
```

Comment:
These helpers are great for reducing the library size. But other helpers are better covered by [Underscore.js](http://underscorejs.org/), [Lodash](https://lodash.com/docs), [functional.js](http://functionaljs.com/) and [Ramda](http://ramdajs.com/) which people already use. We would just duplicate features.